.luarc.json

```diff
@@ -1,12 +1,12 @@
 {
   "$schema": "https://raw.githubusercontent.com/LuaLS/vscode-lua/master/setting/schema.json",
   "Lua.runtime.version": "LuaJIT",
-  "Lua.diagnostics.globals": [
-    "vim",
-  ],
+  "Lua.diagnostics.globals": ["vim"],
   "workspace.library": [
+    "/usr/share/nvim/runtime",
```

Comment:
I see that you are using the only true OS, but sadly this won’t work on MacOS, which also happens to be my main dev environment, so I don’t want it as it is.

However, "$VIMRUNTIME/lua/vim" works for me and seems to be more portable. Could you use that?